README.md

```diff
@@ -0,0 +1,103 @@
+<img src="Supplementary_data/dea_logo_wide.jpg" width="900" alt="Digital Earth Australia logo" />
+
+# DEA Notebooks
+
+[![DOI](https://img.shields.io/badge/DOI-10.26186/145234-0e7fbf.svg)](https://doi.org/10.26186/145234) [![Apache license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![PyPI](https://img.shields.io/pypi/v/dea-tools)](https://pypi.org/project/dea-tools/) [![Notebook testing](https://github.com/GeoscienceAustralia/dea-notebooks/actions/workflows/test_notebooks.yml/badge.svg?branch=develop)](https://github.com/GeoscienceAustralia/dea-notebooks/actions/workflows/test_notebooks.yml)
+
+<br />
+
+This is the DEA Notebooks and DEA Tools repository of [Digital Earth Australia](https://www.dea.ga.gov.au/).
+These can be viewed on the [DEA Knowledge Hub](https://docs.dea.ga.gov.au/) and run on the
+[DEA Sandbox](https://app.sandbox.dea.ga.gov.au/) and the [National Computational Infrastructure (NCI)](https://nci.org.au/).
```

Comment:
I think we should point to our own docs here, as there's not an easy path for users to get started on the NCI without our custom setup. I wouldn't be against pointing to our Sandbox docs here either, but it's less critical than for the NCI.

```suggestion
[DEA Sandbox](https://app.sandbox.dea.ga.gov.au/) and the [National Computational Infrastructure (NCI)](https://docs.dea.ga.gov.au/guides/setup/NCI/README/).
```